### ISSUE TITLE
Please un-nerf the advanced chip capacitors.

### ISSUE THREAD

**Issue by defanivee** [GTNewHorizons/GT-New-Horizons-Modpack#7573]
#### Which modpack version are you using?
2.1.0.4
#
#### If in multiplayer; On which server does this happen? 
All
#
#### What did you try to do, and what did you expect to happen?
I wanna use advanced chip capacitors instead of the normal ones to half the time of production.
#
#### What happened instead? (Attach screenshots if needed)
The advanced capacitors are unworthy to make compared to the normal capacitors. Well, I think advanced chips are great design overall. Upgrading materials leads to lower time consumptions, fine. But I found the advanced capacitors are so expensive even compared with the other advanced chips. **It consumes Os, an extremely rare material with very complicated procedure to get before the T5 rocket, but only produce 8 (=32 normal capacitors ≈24) at a time.** The recipes produce similar amount of products but the normal capacitors are way much cheaper.(Please compare the PBI with the ethylene, and HSSS with tantalum) The other advanced chips are also expensive, but at least they produce 4 times as much products. **Another key thing is that every cuicuit that doesn't require diode needs capacitors. :-( Only do the capacitors prohibited me to use the advanced recipes at the end.**
#
#### What do you suggest instead/what changes do you propose?
The reason that the advanced capacitors are so expensive because you copied the recipe with the lowest efficiency for the normal capacitor. **BUT NO ONE REALLY USE THAT ONE!** It's really expensive and slow to produce. I recommend you to change the products to 24 instead of 8. (or at least 16! After all, the yields of Os is too terrible!)

**Comment by GeicowithaBango** [comment]
the yield of osmium, and the described consumption of it to make the adv smd parts  is intended. we had this discussion at the time when the adv smds were introduced

**Comment by defanivee** [comment]
> the yield of osmium, and the described consumption of it to make the adv smd parts is intended. we had this discussion at the time when the adv smds were introduced

Well, do you think to use more expensive and rarer materials to make the same amount of product a fair trade? capacitors are the most required smds among all of them. Only 1/4 types of circuits don't need capacitors. Well, it doesn't seem like a tech boost, but a nasty food you have to taste.

 Not only does the high expense prevent players from using it in IV. It will decrease the value of crystal circuits. Crystals need more Pt and ethylene, while quantum circuits only need slightly more Cu. Crystals only have already had minimal benefits before, now the price of adv capacitors only kills my desire of using this "more advanced" tech.

**Comment by GeicowithaBango** [comment]
> Not only does the high expense prevent players from using it in IV. It will decrease the value of crystal circuits. Crystals need more Pt and ethylene, while quantum circuits only need slightly more Cu. Crystals only have already had minimal benefits before, now the price of adv capacitors only kills my desire of using this "more advanced" tech. <

you are right on that. if it turns out that we want high circuits being more favorable then the lower tier ones, this should be definitely the case. 
however, this issue was shortly discussed on gtnh discord months ago and as it turns out it’s quite difficult to figure out a proper balance attempt.

**Comment by Prometheus0000** [comment]
> Not only does the high expense prevent players from using it in IV.

They're LuV though? You don't need to use them before that.

**Comment by defanivee** [comment]
> They're LuV though? You don't need to use them before that.

You can say that they are in ZPM. I'm not forced to use it in IV anyway. But I will not use it even if I have a pbi line. Just because of the adv capacitors.

> it turns out it’s quite difficult to figure out a proper balance attempt.

Why isn't buffing capacitors to a normal yield a proper balance strategy? Just look at the other adv smds. They all have a normal yield.

My points are 2.

1. Adv capacitors, the most needed type of smd, only yield 33% more per recipes compared to normal smds. But other adv smds yield 4 times as much. Plus, hsss is the most expensive metal among all smds. 1HSSS+3PBI=1.33Ta+4ethylene? Is that the balance?

2. Don't you think players should get some benefits from a more advanced tech? At least there should be motivations for people to make new things when they're able to build some required machines and have the ability to produce ingredients. So what's my motivation to use adv smds if one recipe yields 8 instead of 24? Time? PBI, Os and HSSS also need time to produce. I would rather build more circuit assemblers if it's so expensive. And the crystals. Crystals had already brought only minimal benefits compared with quantums. With the expensive capacitors required, I just have no reason to use crystals for non-UV circuits after fusion. Small yield per recipe prevents me to use high-tech. It isn't a good design I think.

For the same reason, players who make the PBI line and the whole pt lines in IV should be rewarded for the right to use adv smds. But it seems not to be a reward now... adv smds only add to burdens **JUST BECAUSE OF THE ADV CAPACITORS**.

btw, I'm using adv diodes now. I think all the other adv smds are great.

**Comment by GeicowithaBango** [comment]
btw, as you described crystals are past fusion and we dont want the game to be too ez, right :P

**Comment by defanivee** [comment]
> btw, as you described crystals are past fusion and we dont want the game to be too ez, right :P

You simply ignored all my arguments.

So you think **HARD>BALANCED&REASONABLE**? And advanced tech should make players use more ingredient to craft the same thing?

Based on your opinion, why not make the cost on circuits 6 times more in CAL? It's **hard** as you like. And players **don't get any benefits from advanced techs**. Yes! That's definitely what you want. 💯

**Comment by Dawnot** [comment]
> btw, as you described crystals are past fusion and we dont want the game to be too ez, right :P

What do you mean by WE? Why do you think you can represent all other players? Go and speak to 0lafe if you want the game to become harder :)))
![D)N%FBX8(~C9 Q3T2ZFWAE2](https://user-images.githubusercontent.com/23292848/114530966-0ba35d00-9c7e-11eb-9097-ae38b09a33e7.jpg)

**Comment by GeicowithaBango** [comment]
„we“ as in those people who discussed the asmd when they got introduced. 
i dont know what your problem is now, i answered both arguments of your criticism regarding the adv capacitor. 
You merely repeated them again waiting for a different answer lol

Let me rephrase my comment, since you have access to fusion and its big power yield, its in the games interest to make things more expensive resource-wise since otherwise everything will become dirt cheap -and i mean everything.
 its a bad design attempt to have free power (and later on free resource mining) while every high tier recipe is made to be more favorable then those from the a lower tier.
it may seem not rewarding at all but im afraid thats how it has to be

**Comment by ScriptedPiky** [comment]
> They're LuV though? You don't need to use them before that.

This pretty much summarizes their usage.

While in the original thread when they were proposed (#5928), I had various different values and intentions for them and were meant for late-game usage, as well as crafting early tiered circuits much faster. However, with the cost that is required and the prevalence of cheap SoCs, the latter is entirely irrelevant by said SoCs, and the former is incredibly expensive nonetheless for what amount of resources you input for them; PBI chugging recipes. Also, for reference, one Osmium equals 288 Advanced Capacitors.

Although, I am open to potential ideas and or adjustments of these recipes if they are needed.

**Comment by defanivee** [comment]
I'm really happy to see a receptive participant.

Well, Socs are not important at all for this topic. Anyone will use them anyway in the late game. Socs really make the production convenient.

> Also, for reference, one Osmium equals 288 Advanced Capacitors.

Well, you definitely have seen that equation:

> 1HSSS+3PBI=1.33Ta+4ethylene

And I have to mention that 1Os=12Ta in the recipe of capacitors.

Os is not the only thing that cost in the recipe, it's the most expensive part though. But that doesn't mean that the other ingredients are neglectable.

> Crystals had already brought only minimal benefits compared with quantums. With the expensive capacitors required, I just have no reason to use crystals for non-UV circuits after fusion. Small yield per recipe prevents me to use high-tech. It isn't a good design I think.

> players who make the PBI line and the whole pt lines in IV should be rewarded for the right to use adv smds. But it seems not to be a reward now.

This is the point I mentioned many many times which I don't wanna repeat anymore... ASMDs and crystals don't feel like "advanced tech" now. Recent un-nerf on the wetwares is also based on this point.

As the proposer of asmds, don't you like to see people be willing to use your stuff? The other threes are great and balanced. Their yields and time reduction compensate for a higher price and make me feel rewarding. But adv capacitors do not. So I think we should add the yield to 24 per recipe.

And at last, harder!=better, balanced, more interesting.

**Comment by ScriptedPiky** [comment]
> Well, Socs are not important at all for this topic. Anyone will use them anyway in the late game. Socs really make the production convenient.

Heh, good point. I do remember I wanted them as "large batch crafting," or something like X-SMD makes Y amount of extra circuits compared to J and H recipes, though this may cause complications in the Circuit Assembly Line if over multiplied by 2

> > 1HSSS+3PBI=1.33Ta+4ethylene
> And I have to mention that 1Os=12Ta in the recipe of capacitors.

Huh, I originally didn't know that 1 Os equated to 12 Ta, that's neat to know for resource value!

> Os is not the only thing that cost in the recipe, it's the most expensive part though. But that doesn't mean that the other ingredients are neglectable.

Oh yeah, the pure iridium dust (which is currently a calcium hog) and the type of gas used, along with the looming tungsten changes, there is potential for shifts on how slow or faster & or difficulty creating HSS-S may become.

> > Crystals had already brought only minimal benefits compared with quantums. With the expensive capacitors required, I just have no reason to use crystals for non-UV circuits after fusion. Small yield per recipe prevents me to use high-tech. It isn't a good design I think.

I just took a look and I can agree that they are incredibly expensive compared to Quantum circuits, which take regular SMDs and cost a fraction of the resources for a regular circuit below UV. References for anyone's curiousity: 
![CrystalCircuits1](https://user-images.githubusercontent.com/41557212/114549259-a61b8600-9c15-11eb-9336-7e198330e88e.PNG)
![CrystalCircuits2](https://user-images.githubusercontent.com/41557212/114549262-a6b41c80-9c15-11eb-8703-10082d4bcd16.PNG)

> > players who make the PBI line and the whole pt lines in IV should be rewarded for the right to use adv smds. But it seems not to be a reward now.
> 
> This is the point I mentioned many many times which I don't wanna repeat anymore... ASMDs and crystals don't feel like "advanced tech" now. Recent un-nerf on the wetwares is also based on this point.
> 
> As the proposer of asmds, don't you like to see people be willing to use your stuff? The other threes are great and balanced. Their yields and time reduction compensate for a higher price and make me feel rewarding. But adv capacitors do not. So I think we should add the yield to 24 per recipe.
> 
> And at last, harder!=better, balanced, more interesting.

And to answer all four parts of this, oh I absolutely wish to see them being an "expensive reward" or an "advanced & high tech component used for the utmost special types of machines or tiers." Granted, that vision boiled down to its simplest forms is purely a PBI plastic dump and getting a bunch of specific alloys or metals to make scarce amounts of higher tiered components purely out of a mandatory requirement rather than reward. Thus, I am willing to see different changes or adjustments or even reworks to these things. Their identity of "advanced" is rather simplified by throwing alloys, but what is something that can be done differently? Perhaps more unique materials like fine wires? Maybe bolts of sorts? Perhaps time to get creative!

**Comment by defanivee** [comment]
I haven't thought of any creative ideas at the time. After all, I'm just for un-nerfing adv capacitors at the beginning lol

Just for the current topic, I think there's enough support for discussing the increment of yields if there's no substitute. Well, if someone proposes better suggestions, I will support him.  And maybe crystals also need some buffs? Actually, I think crystals still have some small advantages if adv capacitors have a more reasonable price.

**Comment by GeicowithaBango** [comment]
how would the crystal buff look like, iyo?

**Comment by ScriptedPiky** [comment]
For a buff, the easiest & simplest solution would be to increasing ASMD Capacitor amount from 8 to 24. Maybe buffing or adjusting others if they are needed

Alternatively but requires more discussions and thoughts is increasing the "complexity" and or resource values of what goes into the ASMDs by adding various other bits and looking at the PBI values. The ideal goal would be something difficult but rewarding to achieve for the player regardless of lower-tiered circuits or high-tiered/mandated requirement circuits. Though I am uncertain what materials to use for these recipes.

**Comment by GeicowithaBango** [comment]
If you feel like it’s necessary to change the ratios regarding the amount you should definitely do it. 
and you know what, crystal circuits should also be more favorable to use. 
Just be careful about the balancing the even higher tier circuits, because it would turn out problematic if bioware >wetware>crystal

**Comment by Glease** [comment]
From piky on discord
> Here are some rough material ideas for these little bits; assembly line recipe, crafts 64 each batch, though unsure about time requirement or voltage requirement;
Each recipe requires 720L (5 ingots) of soldering alloy and 1152L (8 ingots) PBI 
Diode: 8 InGaP dust, 16 HSS-S bolts, 32 fine niobium-titanium wire, 32 fine tungsten steel wire
Transistor: 16 V3Ga foils, 16 vibrant alloy bolts, 32 fine HSS-G wire
Capacitor: 32 HSS-E foils, 16 niobium-titanium bolts, 32 nichrome bolts
Resistor: 16 HSS-E foils, 32 YBa2Cu3O7 bolts, 16 naquadah bolts

**Comment by ScriptedPiky** [comment]
Has there been any thoughts so far on whether or not the changes above should be applied?

**Comment by Prometheus0000** [comment]
I'm not 100% clear on the point. To put it clearly (form what I understand): It's not worth making the advanced SMDs for circuits that can use the non-advanced, because they're just so expensive? And the speed increase doesn't matter enough? Or is it just about SMD capacitors for some reason? I don't even know what the recipe is for that.

Personally, I always thought it was weird you could use the advanced ones in earlier circuits. Plus, SoC versions don't use any SMDs. And I never got to LuV (like many people), so I can't really say much about the expense or whatnot.

**Comment by Glease** [comment]
FIY The whole point of SoC is to not use SMD. Lore wise, you have them all integrated into the wafer. Gameplay wise, they are several times harder to make than the non SoC version.

**Comment by Prometheus0000** [comment]
I'd say they use more advanced materials and higher tier voltage than are actually harder to make.

**Comment by ScriptedPiky** [comment]
These were the original values for the 'suggested' changes.
```Diode: 8 InGaP dust, 16 HSS-S bolts, 32 fine niobium-titanium wire, 32 fine tungsten steel wire
Transistor: 16 V3Ga foils, 16 vibrant alloy bolts, 32 fine HSS-G wire
Capacitor: 32 HSS-E foils, 16 niobium-titanium bolts, 32 nichrome bolts
Resistor: 16 HSS-E foils, 32 YBa2Cu3O7 bolts, 16 naquadah bolts```
However, these could be adjusted and or changed to fit better material tiering, though I don't know what kind of material to use in the case of IV-LuV that isn't an EBF time-sink material.

**Comment by GlodBlock** [comment]
what about enriched naquadah? it is unlocked in IV and don't need much time to smelt with the naquadah rework

**Comment by ScriptedPiky** [comment]
Heh, that could work for one of the ASMDs
